### ISSUE TITLE
Inferred release should run after a scrape session is completed.

### ISSUE THREAD

**Issue by ohinds** [Recidiviz/recidiviz-data#405]
Now it runs on a cron schedule no matter whether the scrape finished or not. This is suboptimal because a) we might wait longer than we have to to infer releases if the scrape finishes quickly and b) release is inferred even if the scraper did not complete ingestion, which causes false positives in the release logic.

**Comment by ohinds** [comment]
This should be easy after #974

**Comment by arian487** [comment]
Duplicate of https://github.com/Recidiviz/recidiviz-data/issues/1044
